cmd/kpod/login.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/containers/image/pkg/authentication"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+const defaultServer = "https://registry-1.docker.io"
+
+var (
+	loginFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "password, p",
+			Usage: "Password for registry",
+		},
+		cli.StringFlag{
+			Name:  "username, u",
+			Usage: "Username for registry",
+		},
+	}
+	loginDescription = "Login to a container registry on a specified server.\n" +
+		"The default registry is https://registry-1.docker.io"
```

Comment:
let's remove this default registry. Docker isn't special in kpod :) let's just remember "docker.io" maps to "registry-1.docker.io" for login. I expect people to type `kpod login docker.io` as well as `kpod login myregistry.com:5000`. I don't want Docker to be special

Reply:
I agree, I would rather just have it read registries.conf and default to the primary registry, or just eliminate defaults altogether and force people to specify registry.

Reply:
I like the remove the second line thought here, but don't like eliminating the default.  Having a default registry specified in the conf file would work well.  Typing "kpod login myregistry.com:5000" is going to get old quick